src/middleware/node/get-missing-headers.ts

```diff
@@ -0,0 +1,12 @@
+import { IncomingMessage } from "http";
+
+const WEBHOOK_HEADERS = [
+  "x-github-event",
+  "x-hub-signature-256",
```

Comment:
I intentionally left out the `x-hub-signature` header here, we want to nudge users to use best practices. GHES support is good enough for the new header